web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -621,10 +631,32 @@
     get siblings() {
       try {
         const wrapper = this.closest(this.getAttribute('interdependence-wrapper'));
-        return [...wrapper?.querySelectorAll('cel-input-date-time, cel-input-date, cel-input-time') || []];
+        this.#siblings = [...wrapper?.querySelectorAll('cel-input-date-time, cel-input-date, cel-input-time') || []];

```

Comment:
if anything in the dom tree is changing nearby you may loose control over which elements needs to be observed. I think it would be more safe, if each element desides for itself which other elements it wants to observe.